packages/geocoder/src/geocoders/abstract-geocoder.ts

```diff
@@ -125,10 +132,10 @@ export default class Geocoder {
    * Do a reverse-geocode, i.e. get address information and attributes given a
    * GPS coordinate.
    */
-  reverse(query: ReverseQuery): Promise<MultiGeocoderResponse> {
+  reverse(query: ReverseQuery): Promise<MultiGeocoderResponse | SingleGeocoderResponse> {
```

Comment:
Maybe create an alias `type SingleOrMultiGeocoderResponse = MultiGeocoderResponse | SingleGeocoderResponse` and reuse? (~5 occurrences)